### PR TITLE
add fusion methods

### PR DESCRIPTION
**Pull request by jharenza**
#### Purpose

_Briefly describe your changes here._

- add FusionAnnotator to workflow
- update Fusion methods according to fusion PRs; analysis [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering)

#### Issue

_What GitHub issue does your pull request address?_

#### Pull review checklist

The PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/jaclyn-taroni/OpenPBTA-manuscript/blob/master/USAGE.md#figures)

### CODE REVIEW

File: content/03.methods.md [hunk 1]
```diff
@@ -138,18 +138,19 @@ For both these tools we used aligned BAM and chimeric SAM files from STAR as inp
 We ran STAR-Fusion with default parameters and annotated all fusion calls with GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz provided in the STAR-fusion release. 
 For Arriba, we used a blacklist file (blacklist_hg38_GRCh38_2018-11-04.tsv.gz) from the Arriba release tarballs to remove recurrent fusion artifacts and transcripts present in healthy tissue.
 We also provided Arriba with strandedness information or set it to auto-detection for polyA samples.
-The RNA expression and fusion workflows can be found in the [KidsFirst GitHub repository](https://github.com/kids-first/kf-rnaseq-workflow).
+We used [FusionAnnotator](https://github.com/FusionAnnotator/FusionAnnotator) to harmonize Arriba fusion annotations with those of STAR-Fusion.
```
Comment (jaclyn-taroni) [inline content/03.methods.md]: Can you add some more context here? I know one of these tools automatically does this for you, but I don't remember which one.

File: content/03.methods.md [hunk 2]
```diff
@@ -138,18 +138,19 @@ For both these tools we used aligned BAM and chimeric SAM files from STAR as inp
...
-We also removed fusions if the 5\` or 3\` ends were the same gene, and these were tagged as non-canonical splicing or duplication.
-We used a list of curated fusion calls for each histology to capture each occurrence of the fusion as a putative driver fusion.
-We prioritized a union of fusion calls as true calls if the fused genes were detected by both callers, the same fusion was recurrent in histology (>2 samples) or the fusion was specific to the broad histology. 
-We annotated putative driver fusions and prioritized fusions lists with kinases, oncogenic, tumor suppressor, transcription factor, fused genes and known TCGA fusions from curated [datasheets](https://github.com/d3b-center/fusion_filtering_pipeline/tree/master/references).
-We also added chimerDB [@doi:10.1093/nar/gkw1083] annotations to both driver and prioritized fusion list.
+We performed artifact filtering and additional annotation on fusion calls to prioritize putative oncogenic fusions.
+Briefly, we considered all inframe and frameshift fusion calls with 1 or more junction reads and fused genes expressed with TPM greater than one to be true calls.
```
Comment (jaclyn-taroni) [inline content/03.methods.md]: To be consistent:
```suggestion
Briefly, we considered all inframe and frameshift fusion calls with 1 or more junction reads and fused genes expressed with TPM greater than 1 to be true calls.
```

File: content/03.methods.md [hunk 3]
```diff
@@ -138,18 +138,19 @@ For both these tools we used aligned BAM and chimeric SAM files from STAR as inp
...
+If a fusion call had large number of spanning fragment reads compared to junction reads (spanning fragment minus junction read greater than ten).
+We prioritized a union of fusion calls as true calls if the fused genes were detected by both callers, the same fusion was recurrent within a `broad_histology` (>2 samples) or the fusion was specific to the `broad_histology`. 
+If either 5' or 3' genes fused to more than five different genes within a sample, we removed these calls as a potential false positive.
```
Comment (jaclyn-taroni) [inline content/03.methods.md]:
```suggestion
If either 5' or 3' genes fused to more than five different genes within a sample, we removed these calls as potential false positives.
```

File: content/03.methods.md [hunk 4]
```diff
@@ -138,18 +138,19 @@ For both these tools we used aligned BAM and chimeric SAM files from STAR as inp
...
-We used a list of curated fusion calls for each histology to capture each occurrence of the fusion as a putative driver fusion.
-We prioritized a union of fusion calls as true calls if the fused genes were detected by both callers, the same fusion was recurrent in histology (>2 samples) or the fusion was specific to the broad histology. 
-We annotated putative driver fusions and prioritized fusions lists with kinases, oncogenic, tumor suppressor, transcription factor, fused genes and known TCGA fusions from curated [datasheets](https://github.com/d3b-center/fusion_filtering_pipeline/tree/master/references).
-We also added chimerDB [@doi:10.1093/nar/gkw1083] annotations to both driver and prioritized fusion list.
+We performed artifact filtering and additional annotation on fusion calls to prioritize putative oncogenic fusions.
+Briefly, we considered all inframe and frameshift fusion calls with 1 or more junction reads and fused genes expressed with TPM greater than one to be true calls.
+If a fusion call had large number of spanning fragment reads compared to junction reads (spanning fragment minus junction read greater than ten).
```
Comment (jaclyn-taroni) [inline content/03.methods.md]: I think you are missing the second half of the sentence here.

File: content/03.methods.md (hunk 1 above)
Comment (jharenza) [reply content/03.methods.md]: done

File: content/03.methods.md (hunk 4 above)
Comment (jharenza) [reply content/03.methods.md]: whoops, fixed, thank you!

**Pull request merged**